src/libopensc/opensc.h

```diff
@@ -658,7 +658,7 @@ struct sc_card_operations {
 	 * @see sc_read_binary()
 	 */
 	int (*read_binary)(struct sc_card *card, unsigned int idx,
-			u8 * buf, size_t count, unsigned long flags);
+			u8 * buf, size_t count, unsigned long *flags);
```

Comment:
Suggestion: If you would document `flags` as optional argument, then the card driver should check if flags is not NULL before setting it to `SC_FILE_COMPRESSED`. If used this way, all card drivers that currently ignore `flags` don't need to be touched with this PR since they can continue to pass `0` as `flags` to `sc_read_binary` (where `0` is what's the definition of `NULL`).

Reply:
We should update the doc text above to reflect this change. Now it says `currently not used`. So replacing with `optional` might be enough.